tests/integration-tests/tests/tags/test_tag_propagation.py

```diff
@@ -25,12 +26,19 @@
     get_main_stack_tags,
     get_shared_volume_tags,
 )
+from time_utils import minutes, seconds
+
+
+def get_pcluster_version():
```

Comment:
Let's check if we can reuse the function from
`from tests.common.utils import get_installed_parallelcluster_version` and remove this function

Reply:
Done!